doc/api/stream.markdown

```diff
@@ -181,6 +209,9 @@ readable.on('data', function(chunk) {
   console.log('got %d bytes of data', chunk.length);
 });
 ```
+Note that the `readable` event should not be used together with `data`
+because the assigning the latter switches the stream into "flowing" mode,
```

Comment:
No, it doesn't contradict it but the word is a bit clumsy and unnecessary with 167-168 in there. I'll just pull this back out.